portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFolderLocalServiceImpl.java

```diff
@@ -1130,7 +1145,7 @@ public void updateLastPostDate(long folderId, Date lastPostDate)
 			return;
 		}
 
-		dlFolder.setModifiedDate(dlFolder.getModifiedDate());
+		dlFolder.setModifiedDate(lastPostDate);
```

Comment:
why is this change needed? It's weird that a method name updateLastPostDate changes the modifieddate, isn't it?

Reply:
While I agree that the method name is strange, if you take a look at the code was previously written, the modified date wasn't being modified at all in that method (it just take its value and applied it again).

I don't know the differences between the modified date and the last post date, but I can confirm that, regarding the folder, the date shown there is indeed the modified date.

Since even one of the arguments sent when the function is called is the modified date, in my opinion the correction is fine, since the intended date to be modified is indeed the modified date, not the last post date.